SourceryRuntime/Sources/Definable.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Describes that the object is defined in a context of some `Type`
+public protocol Defined: class {
```

Comment:
We already have `Definition` protocol, we need to use just one to avoid confusion.